instructions-for-devs.md

```diff
@@ -0,0 +1,206 @@
+This document contains `xud` setup instructions for developers and everyone with a need for more fine-grained control when running `xud`.
+
+## 1. Install
+
+First, clone the repository from GitHub and install dependencies. Make sure to have minimum `nodejs` v8.11.3 installed. If not the case, checkout [the official instructions](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions). 
+
+Developers & Testers:
+```bash
+git clone https://github.com/ExchangeUnion/xud
+cd xud
+npm install
+```
+For everyone else, we recommend using the [latest release](https://github.com/ExchangeUnion/xud/releases), below an example using the `1.0.0-prealpha.1` release tag:
+```bash
+git clone https://github.com/ExchangeUnion/xud 
+cd xud
+git checkout tags/1.0.0-prealpha.1 -b 1.0.0-prealpha.1
+npm install
+```
+
+
+`xud` uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You will have to install one of those and create a user *xud* and a database *xud* and grant all permissions for the new database to the *xud* user.
+
+
+## 2. Start `xud`
+
+Open a new terminal and launch a `xud` process
+
+```bash
+~/xud/bin $ ./xud
+2018-6-3 15:14:27 [GLOBAL] info: config loaded
+2018-6-3 15:14:27 [DB] info: connected to database. host:localhost port:3306 database:xud
+2018-6-3 15:14:28 [P2P] info: pool server listening on 0.0.0.0:8885
+2018-6-3 15:14:28 [RPC] info: GRPC server listening on port 8886
+2018-6-3 15:14:28 [RPC] info: gRPC Web API proxy listening on port 8080
+```
+
+Use `./xucli --help` to get up-to-date, optional command line arguments to override defaults and settings in the [configuration](#configuration-optional) file for a specific `xud` instance
+
+```bash
+~/xud/bin $ ./xud --help
+Options:
+  --help              Show help                                        [boolean]
+  --version           Show version number                              [boolean]
+  --xudir, -x         Data directory for xud                            [string]
+  --db.database       SQL database name                                 [string]
+  --db.host           Hostname for SQL database                         [string]
+  --db.port           Port for SQL database                             [number]
+  --db.username       User for SQL database                             [string]
+  --lnd.certpath      Path to the SSL certificate for lnd               [string]
+  --lnd.disable       Disable lnd integration                          [boolean]
+  --lnd.host          Host of the lnd gRPC interface                    [string]
+  --lnd.macaroonpath  Path of the admin macaroon for lnd                [string]
+  --lnd.port          Port of the lnd gRPC interface                    [number]
+  --p2p.listen        Listen for incoming peers                        [boolean]
+  --p2p.port, -p      Port to listen for incoming peers                 [number]
+  --raiden.disable    Disable raiden integration                       [boolean]
```

Comment:
suggest changing the default to false --raiden.disable

Reply:
It was. Somebody changed that. Acceptable for now though, we'll switch that to false once the lnd/raiden integration actually works.